en/getting-started/install/index.md

```diff
@@ -0,0 +1,70 @@
+# Install and Launch
+
+Starting from v2.3.2, Cocos Creator is now equipped with a new Dashboard system, which allows you to upgrade and manage multiple versions of the engine and projects at the same time! It also integrates a unified project management and creation panel, so that you can use different versions of the engine to develop projects at the same time!
+
+![Dashboard](index/dashboard-editor.png)
+
+## Download Dashboard
+
+You can get the Dashboard installation package by visiting the download link on [Cocos Creator Product Page](https://www.cocos.com/en/creator).
+
+Double-click the installation package after downloading.
+
+### Installation Instructions for Windows Version
+
+The installer for Windows is an `.exe` executable file, usually named **CocosDashboard-vX.X.X-win32-20XXXXXX.exe**, where **vX.X.X** is the version number of Cocos Dashboard, such as v1.0.11. The string of numbers after it is the version date number.
+
+> **Note**.
+> 1. If the version number installed on the current PC is the same as the version number of the installation package, the installation package with the same version number cannot be overwritten automatically, and you need to uninstall the previous version before you can continue the installation.
```

Comment:
```suggestion
> 1. If the version number installed on the current PC is the same as the version number of the installation package, the installation package with the same version number cannot be overwritten automatically, and it will be necessary to uninstall the previous version before continuing the installation.
```